### PR TITLE
Add a build status badge to the readme.

### PR DESCRIPTION
**Pull request by xxxserxxx**
This just adds a github CI build status badge to the README. Does it need to be called out in the changelog?

# Checklist

If applicable:
- [ ] I have updated `CHANGELOG.md`
- [ ] I have updated the documentation (README.md, docs/, demos/)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 - [Disclaimer](#disclaimer)
 - [Introduction](#introduction)
-- [Status](#status)
+- [Status](#status) ![](https://github.com/martinvonz/jj/workflows/build/badge.svg)
```
Comment (martinvonz) [inline README.md:5]: This heading is project status, not build status, so I don't think this belongs here. Do you want to instead add a new section with various badges as @arxanas did for https://github.com/arxanas/git-branchless?

Comment (xxxserxxx) [reply README.md:5]: Do you have any preferences about location? Would you like to be patterned after git-branchless?

Comment (martinvonz) [reply README.md:5]: I don't have much preference, so following git-branchless seems best, unless you have some preference.

Comment (xxxserxxx) [reply README.md:5]: Also, as I doubt the google-cla bot will respond: every link to sign the Google CLA takes me to a Google sign-in page. Is there a way to sign without having a Google account?

Comment (martinvonz) [reply README.md:5]: I just checked our internal docs and they say that you need to have a Google account to sign it. I can take over the PR if you don't want to register for a Google account.

Comment (arxanas) [reply README.md:5]: There's probably no need for a table in this case, like in git-branchless. I've seen many projects just put the status badges as the very first line in the README (even above any headers), which looked fine to me.

**Comment by martinvonz** [comment]
With #805 merged, feel free to add yourself to CHANGELOG.md

File: README.md (hunk 1 above)
Comment (xxxserxxx) [reply README.md]: In the updated PR, I moved the tags to the top (but under the project title), and added a number of other badges; feel free to trim out whatever seems excessive.

In the commit, I explicitly release all rights on the changes. If that's insufficient, please feel free to take over the PR. This seems like a lot of conversation considering the scope of the "work." :-)

Comment (martinvonz) [reply README.md]: I'd be surprised if Google thinks that's enough, so I just created a copy of this PR as #822. Thanks!

**Comment by martinvonz** [comment]
Replaced by #822

**Pull request closed without merging**